ras_party/controllers/enrolments_controller.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from uuid import UUID
+
+import structlog
+from flask import session
+from sqlalchemy.orm.exc import NoResultFound
+
+from ras_party.controllers.queries import (
+    query_enrolments_by_parameters,
+    query_respondent_by_party_uuid,
+)
+from ras_party.models.models import Enrolment
+from ras_party.support.session_decorator import with_query_only_db_session
+
+logger = structlog.wrap_logger(logging.getLogger(__name__))
+
+
+@with_query_only_db_session
+def enrolments_by_parameters(
+    session: session, party_uuid: UUID = None, business_id: UUID = None, survey_id: UUID = None, status: int = None
+) -> list[Enrolment]:
+    """
+    returns a list of Enrolments based on provided parameters
+    """
+
+    if party_uuid:
```

Comment:
I don't want to join the tables, so decided to do it this way. Enrolment does not have the respondent uuid which is used in every use I can see

Reply:
As discussed off-line and commented below, for the use cases this PR is intending to address/improve this is fine. However, we should either tweak the `/enrolments` API so the respondent uuid is not optional (e.g. /enrolments/<RESPONDENT_UUID>` or join the tables to return the uuid if it's not included as a parameter.

The former is probably fine for Frontstage use cases, but the later may be needed for Rops use cases.